docs/pages/application-access/okta/reference.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Okta Service Reference Documentation
+description: Configuration and CLI reference documentation for Teleport Okta service.
+---
+
+## Configuration
+
+(!docs/pages/includes/backup-warning.mdx!)
+
+The following snippet shows the full YAML configuration of an Okta Service
+appearing in the `teleport.yaml` configuration file:
+
+```yaml
+(!docs/pages/includes/config-reference/okta-service.yaml!)
+```
+
+## Okta import rule resources
+
+Full YAML spec of Okta import rule resources managed by `tctl` resource commands:
+
+```yaml
+kind: okta_import_rule
+version: v1
+metadata:
+  # Okta import rule name.
+  name: test-rule
+  # Okta import rule description.
+  description: "Okta import rule for admins"
+spec:
+  # Mappings describes a list of match directives that describe what lavels to
+  # apply to specific Okta groups and applications that will be synchronized
+  # into Teleport.
+  mappings:
+  - match:
+    # Individual application IDs can be used to create matches. Imported Okta
+    # applications will be matched based on the app IDs listed here.
+    - app_ids: ["app1", "app2"]
+    # These lablels will be added to matching applications.
+    add_labels:
+      app_label: app_label_value
+  - match:
+    # Similarly, group IDs can be used to create matches.
+    - group_ids: ["group1", "group2"]
+    # These labels will be added to the groups.
+    add_labels:
+      label1: value1
+```
+
+You can create a new `okta_import_rule` resource by running the following commands, which
+assume that you have created a YAML file called `okta-import-rule.yaml` with your configuration:
+
+<ScopedBlock scope={["enterprise"]}>
+
+```code
+# Log in to your cluster with tsh so you can use tctl from your local machine.
+# You can also run tctl on your Auth Service host without running "tsh login"
+# first.
+$ tsh login --proxy=teleport.example.com --user=myuser
+# Create the resource
+$ tctl create -f okta-import-rule.yaml
+```
+
+</ScopedBlock>
+
+## Okta assignment resources
+
+Full YAML spec of Okta assignment resources queried by `tctl` resource commands:
+
+```yaml
+kind: okta_assignment
+version: v1
+metadata:
+  # The name of the Okta assignment.
+  name: test-assignment
+spec:
+  # The user that the Okta assignment is granting access for.
+  user: test-user@test.user
+  # The list of targets to grant access to.
+  targets:
+  # An application target.
+  - type: 1
+    id: "123456"
+  # A group target.
+  - type: 2
+    id: "234567"
+  # The current status of the Okta assignment.
+  status: 1
```

Comment:
What are the valid statuses? `status: 1` doesn't really give any clue as to what this is actually for.